src/NewTools-SettingsBrowser-Tests/StSettingsBrowserTest.class.st

```diff
@@ -13,9 +13,8 @@ Class {
 { #category : 'running' }
 StSettingsBrowserTest >> application [
 
-	^ StSettingsApplication new
-		  useBackend: #Morphic;
-		  yourself
+	^ StPharoApplication current
```

Comment:
I am really not sure, but shouldn't we do something like `StSettingBrowser new application`? Like this, if the app changes one day, it'll still be the right one? (As I said, I'm really not sure here)